code/game/machinery/doors/door.dm

```diff
@@ -289,7 +284,7 @@
 			L.adjustBruteLoss(DOOR_CRUSH_DAMAGE)
 			L.emote("scream")
 			L.Weaken(5)
-		else if(ismonkey(L)) //For monkeys
+		else if(issmall(L)) //For monkeys

```

Comment:
this entire else if is redundant, if monkeys are a subtype of humans with human emote code, they can scream too